docs/FAQ.md

```diff
@@ -0,0 +1,58 @@
+# Frequently asked questions
+
+### Why does my branch not move to the new commit after `jj new/commit`?
+
+Jujutsu's branches are akin to Git Tags, they always point to a single commit.
+To move them, use `jj branch set`. 
+
+### Where is my commit, why is it not visible in `jj log`? 
+
+If you don't find it with `jj log -r 'all()'`, it may be time to set the config
+`revsets.log` to something more matching.
+
+See [revsets], [templates] and [config] for guidance.
+
+### Is there something like `git rebase`? 
+
+For now its recommended to rebase the commit individually with 
+`jj rebase -s $changeid -d $new-parent`.
+
+In the future there may be a `histedit` command, which alleviates that.
+
+
+### How do I partially add changes from a file? 
+
+With Jujutsu you never partially add hunks, as your always amending the 
+`@` commit. You need to split it, as the `@` may already be complete.
+
+This is easy doable with `jj split your-file`
+
+
+### How can I keep my scratch files in the repository?
+
+You can keep your notes and other scratch files in the repository, if you add 
+a wildcard pattern to the `gitignore`. Something like `*.scratch` or 
+`*.scratchpad` should do, after that rename the files you want to keep around
+to match the pattern.
+
+### How can I keep my Git-like workflow?
+
+In contrast to Git, we suggest keeping the branch pointing at the first commit.
```

Comment:
I think we shouldn't recommend this. It's a nice trick, but it comes with its problems. For example, pushing branches becomes confusing if you do this.

I think we should delete or rethink this question.

Reply:
I think people need a crutch, even if its not our ideal solution. If you have a better workflow to add here, feel free to suggest it.

Reply:
I think the point that `jj git push --all` won't work is too big a drawback. I'm not sure what to suggest, because there isn't really a Git-like flow, other than suggesting a combination of `jj commit && jj branch set $branch -r @-`

Reply:
I know, as I've written the [github](https://github.com/martinvonz/jj/blob/main/docs/github.md) doc.

Reply:
I thought about rewriting the section, but it's easier to just drop it.